mitodl/mit-open
# Homepage "Personalize"

### What are the relevant tickets?
- https://github.com/mitodl/hq/issues/4363

### Description (What does it do?)
Adds the "Personalize your Journey" section of the homepage.

### Screenshots (if appropriate):

<img width="1894" alt="Screenshot 2024-06-13 at 12 13 12 PM" src="https://github.com/mitodl/mit-open/assets/9010790/f651e3d8-5b21-498e-827a-71fbf1cfc742">
<img width="892" alt="Screenshot 2024-06-13 at 12 13 18 PM" src="https://github.com/mitodl/mit-open/assets/9010790/e55ebbdd-029a-40cc-a2ee-ea34feb5883c">
<img width="779" alt="Screenshot 2024-06-13 at 12 43 06 PM" src="https://github.com/mitodl/mit-open/assets/9010790/d7603fe5-cf1f-4eb6-8dca-6aceb97c16f0">



### How can this be tested?
View the homepage...
1. On desktop vs mobile vs other widths. It should look like screenshots above.
2. As an authenticated or unauthenticated user:
    - authenticated user the button link goes to dashboard
    - unauthenticated, the button link goes to signup flow
3. View the storybook pages for buttons. They should still look the same. I made two signficant changes to buttons:
    - They are no longer fixed height. Now they can wrap. But when there is a single line of text, all small / medium / large variants should still be 32px / 40px / 48px.
    - I added an (optional, default false) `responsive` prop. When it's true, buttons will move down one size on mobile.
